crosstests/google-javalite/build.gradle.kts

```diff
@@ -0,0 +1,44 @@
+import com.diffplug.gradle.spotless.SpotlessExtension
```

Comment:
We were generating java lite code under this directory, but nothing was configured to build it. Added a build file although there may be additional work to do if we want the crosstests to run on the lite code.